src/services/admin/mentor.service.ts

```diff
@@ -36,3 +49,85 @@ export const updateMentorStatus = async (
     throw new Error('Error updating the mentor status')
   }
 }
+
+export const getAllMentors = async (
+  status: ApplicationStatus
+): Promise<{
+  statusCode: number
+  mentors?: AllMentorsArray[]
+  message: string
+}> => {
+  try {
+    const mentorRepository = dataSource.getRepository(Mentor)
+
+    const mentorsList: Mentor[] = await mentorRepository.find({
+      where: { state: status },
```

Comment:
Status is optional, by default it will return all the mentors if the status is not present